src/System.Net.Security/src/System/Net/SecureProtocols/_SslStream.cs

```diff
@@ -207,7 +207,6 @@ private void ProcessWrite(byte[] buffer, int offset, int count)
                 throw new NotSupportedException(SR.Format(SR.net_io_invalidnestedcall, "Write", "write"));
             }
 
-            bool failed = false;
```

Comment:
Nit: the comment above about asyncRequest==null looks out of date... there is no asyncRequest here.